docs/tutorials/AddRemark.md

```diff
@@ -25,7 +25,7 @@ For now, let’s keep `RemarkCommand` as simple as possible and print some outpu
 ``` java
 package seedu.address.logic.commands;
 
-import seedu.address.model.Model;
+import model.bookface.Model;
```

Comment:
should line 28 be `import bookface.model.Model;` instead?

Reply:
oh wait nvm this is the tutorial doc so i don't think it matters

Reply:
i didnt know that this doc was inside (because checkstyle only checks for /test and /main folders only)

Reply:
@sprintaway ohh okay gotcha, I'll not bother too much with the md files